.buildkite/tests/values_capabilities_proxy_test.py

```diff
@@ -1,19 +1,21 @@
-import pytest
 import time
+
+import pytest
+
 from config import NAMESPACE
-from fixtures import setup_cluster, kube_client, cleanup_agent_from_cluster
```

Comment:
Guessing this was removed by mistake... (it is required)

Reply:
Oops.. linter got the best of me